src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/OffenderService.kt

```diff
@@ -101,12 +105,22 @@ class OffenderService(
     offender.assessments.add(assessment)
     offenderRepository.save(offender)
 
+    val changes = mapOf(
+      "NOMS-ID" to prisoner.prisonerNumber,
+      "PRISONER_HDCED" to prisoner.homeDetentionCurfewEligibilityDate.format(DateTimeFormatter.ISO_DATE),
```

Comment:
can we change all the fields to be camelcase- it'll be easier if we ever need to read them out and mean we don't have inconsistencies between `_` and `-`. 
Would also be good to be consistent with naming: `prisonNumber`,  `homeDetentionCurfewEligibilityDate` etc..